spec/requests/event_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Event' do
+  describe 'POST /events' do
```

Comment:
Please split positive and negative test cases into defferent context blocks

Reply:
Splitted into different context blocks